### DISCUSSION TITLE
How to modify GitHub release notes?

### DISCUSSION THREAD

**Discussion by bartekpacia** [jreleaser/jreleaser#1701] [Q&A] [answered]
Hey, thanks for this awesome project!

I have a question: how can I modify the GitHub release notes?

```kotlin
jreleaser {

    project {
        // ...
    }

    distributions {
        create("maestro") {
            stereotype.set(Stereotype.CLI)

            executable {
                name.set("maestro")
            }

            artifact {
                setPath("build/distributions/maestro.zip")
            }
            
            release {
                github {
                    repoOwner.set("mobile-dev-inc")
                    name.set("maestro")
                    tagName.set("cli-$CLI_VERSION")
                    releaseName.set("CLI $CLI_VERSION")
                    // I want to modify release notes
                    overwrite.set(true)
                }
            }
        }
    }
}
```

My project is open source, [here's my config](https://github.com/mobile-dev-inc/maestro/blob/d453545b72b3068167b39bab24e1c1f2b281495a/maestro-cli/build.gradle.kts#L91-L150).

**Comment by bartekpacia** [comment]
Found the answer by reading this discussion https://github.com/jreleaser/jreleaser/discussions/1186

**Comment by aalmiray** [comment, marked as answer]
Use the `changelog` section of the configured releaser (in this case, `github`) to tweak how release notes should be generated https://jreleaser.org/guide/latest/reference/release/changelog.html

**Comment by bartekpacia** [comment]
~~Hmm, [I specified `changelog` explicitly](https://github.com/mobile-dev-inc/maestro/blob/702b184d6be087378245d4818910b2652de7048d/maestro-cli/build.gradle.kts#L127-L137), but JReleaser didn't use it. [It still generated the default one](https://github.com/mobile-dev-inc/maestro/releases/tag/cli-1.37.6).~~ [solution](https://github.com/jreleaser/jreleaser/discussions/1186#discussioncomment-4827330)
